src/index.ts

```diff
@@ -1,6 +1,26 @@
+import {
+  BUILDER_ITEMS_PREFIX,
+  LIBRARY_ITEMS_PREFIX,
+  PLAYER_ITEMS_PREFIX,
+  PROD_BUILDER_HOST,
+  PROD_LIBRARY_HOST,
+  PROD_PLAYER_HOST,
+} from './constants';
+import { Context } from './constants/context';
+import { ClientHostManager } from './interfaces/shortLink';
+
 export * from './utils';
 export * from './constants';
 export * from './interfaces';
 export * from './services';
 export * from './config';
 export * from './types';
+
+// Instanciate and add prefix and default host for the diffrent contexts.
+ClientHostManager.getInstance()
+  .addPrefix(Context.Builder, BUILDER_ITEMS_PREFIX)
+  .addPrefix(Context.Library, LIBRARY_ITEMS_PREFIX)
+  .addPrefix(Context.Player, PLAYER_ITEMS_PREFIX)
+  .addHost(Context.Builder, new URL(PROD_BUILDER_HOST))
+  .addHost(Context.Library, new URL(PROD_LIBRARY_HOST))
+  .addHost(Context.Player, new URL(PROD_PLAYER_HOST));
```

Comment:
Is it intended to be instanciated in the SDK ?

Reply:
As I understand our discussion in Slack, yes. I don't know if that's what you had in mind too. The advantage is that it avoids having to instantiate it in every project. I don't know whether it's a good idea to do it the way I did or not.